doc/install_ubuntu.txt

```diff
@@ -1,30 +1,33 @@
-:orphan:
-
 .. _install_ubuntu:
 
-Easy Installation of an Optimized Theano on Current Ubuntu
-==========================================================
 
-For NVIDIA Jetson TX1 embedded platform:
+Ubuntu Installation Instructions
+================================
 
-.. code-block:: bash
+There are two ways of getting the dependencies for Theano. You can either grab them
+as system packages or let pip do everything for you. 
 
-    sudo apt-get install python-numpy python-scipy python-dev python-pip python-nose g++ libblas-dev git
-    pip install --upgrade --no-deps git+git://github.com/Theano/Theano.git --user  # Need Theano 0.8(not yet released) or more recent
+There are also two installation modes, the stable version and the bleeding-edge most recent version.
+
+Requirements through System Packages
```

Comment:
I think it's best not to split the installation in requirements and installation. Let's just use the include directive to add as last step the installation.